firmware/module.json

```diff
@@ -2,10 +2,12 @@
   "name": "mbFirmata",
   "version": "1.0.0",
   "description": "The micro:bit runtime common abstraction with examples.",
-  "license": "MIT",
+  "license": "Apache-2.0",
   "dependencies": {
-    "microbit": "lancaster-university/microbit#v2.1.1"
+    "codal-microbit": "microbit-foundation/codal-microbit"
```

Comment:
Could we switch this and the previous dep to both be targetDependencies so we can change which dal we want based on the target.

Reply:
Did you intend to change the license to Apache 2.0? Even if Codal is Apache 2.0, micro:bit Firmata can be MIT; they are compatible licenses.

Reply:
Hi @jhmaloney. Just FYI, Codal is licensed as MIT...

Reply:
Not intentionally, possibly defaulted to it when I `yotta init`'d the directory. I'll correct that in the next commit!

Reply:
Good to know!